.werft/build.yaml

```diff
@@ -89,15 +80,6 @@ pod:
       mountPath: /mnt/secrets/monitoring-satellite-stackdriver-credentials
     - name: monitoring-satellite-preview-token
       mountPath: /mnt/secrets/monitoring-satellite-preview-token
-    - name: gcp-sa
-      mountPath: /mnt/secrets/gcp-sa
-      readOnly: true
-    - name: gcp-sa-release
```

Comment:
The secret is still used blow, let's not delete it:
https://github.com/gitpod-io/gitpod/blob/a365a2ade31ca94477484df214f3f0cf3b0ad7e6/.werft/jobs/build/build-and-publish.ts#L46